src/views/RecIM/views/Admin/index.jsx

```diff
@@ -163,6 +217,26 @@ const Admin = () => {
           <i>"Competition reveals character"</i>
         </Typography>
       </Grid>
+      <Grid item xs={3} textAlign={'right'}>
+        <IconButton onClick={handleAdminMenuOpen} sx={{ mr: '1rem' }}>
+          <SettingsIcon
+            fontSize="large"
+            sx={
+              openAdminMenu && {
+                animation: 'spin 0.2s linear ',
+                '@keyframes spin': {
+                  '0%': {
+                    transform: 'rotate(0deg)',
+                  },
+                  '100%': {
+                    transform: 'rotate(120deg)',
+                  },
+                },
```

Comment:
Creative :)

Reply:
Thanks, I just copied this from the recim home page :laughing: